### PR TITLE
Improve the exec example.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

File: examples/pom.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (yue9944882) [inline examples/pom.xml]: nit: minimum change

Comment (brendandburns) [reply examples/pom.xml]: done.

**Comment by brendandburns** [comment]
@yue9944882 please take another look.

**Pull request merged**